### PR TITLE
Fix front end tests

### LINKED ISSUE #2640: Front end tests failing
## What went wrong? 😲
 

`yarn test`

and..

```
    Details:

    /Users/m2/GitHub/open-msupply/client/node_modules/@mui/material/styles/ThemeProvider.js:3
    import _extends from "@babel/runtime/helpers/esm/extends";
    ^^^^^^

    SyntaxError: Cannot use import statement outside a module
```
## Expected behaviour 🤔

😄 

## How to Reproduce 🔨

Steps to reproduce the behaviour:

`yarn test`

## Your environment 🌱

 - Version: 
 - Platform:
   - [ ] android (tablet)
   - [ ] browser (extra points if you tell us which one)
   - [ ] desktop (windows)
   - [ ] desktop (macOS)
   - [ ] server (windows)

### PR DESCRIPTION
**Pull request by mark-prins**
Fixes #2640 

# 👩🏻‍💻 What does this PR do? 
 
Fixes the tests - two issues due to the plugin framework changes:
- The syntax of the import of mui ThemeProvider
- changes to i18n which weren't re-rendering the language as it should have been

# 🧪 How has/should this change been tested? 

Running tests with `yarn test` - all will pass now.

## 📃 Documentation

_No user facing changes_

### CODE REVIEW

File: client/packages/common/src/intl/currency/currency.test.tsx [hunk 1]
```diff
@@ -42,12 +43,18 @@ describe('currency formatting - en', () => {
 });
 
 describe('currency formatting - fr', () => {
+  beforeAll(() => {
+    const { result: intlUtils } = renderHookWithProvider(useIntlUtils);
+    intlUtils.current.changeLanguage('fr');
```
Comment (clemens-msupply) [inline client/packages/common/src/intl/currency/currency.test.tsx:48]: I get a warning that this needs to be wrapped in act() from `import { act } from 'react-dom/test-utils';`
```suggestion
    act(() => {
      intlUtils.current.changeLanguage('fr');
    });
```
afterwards it all runs fine

File: client/packages/common/src/intl/currency/currency.test.tsx [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import { renderHookWithProvider } from '@common/utils';
 import { useCurrency } from './currency';
+import { useIntlUtils } from '../utils';
```
Comment (clemens-msupply) [inline client/packages/common/src/intl/currency/currency.test.tsx:3]:
```suggestion
import { useIntlUtils } from '../utils';
import { act } from 'react-dom/test-utils';
```

**Review by clemens-msupply — APPROVED** [review]
Works great thanks! There was one warning and I put in a fix as a suggestion...

**Comment by mark-prins** [comment]
thanks @clemens-msupply - have added the suggestion 🙏

**Pull request merged**